Source/GameBaseFramework/Public/Interaction/Abilities/GBFGameplayAbilityTargetActor_Interact.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "Abilities/GameplayAbilityTargetActor_Trace.h"
+
+#include "GBFGameplayAbilityTargetActor_Interact.generated.h"
+
+/** Intermediate base class for all interaction target actors. */
+UCLASS( Blueprintable )
+class GAMEBASEFRAMEWORK_API AGBFGameplayAbilityTargetActor_Interact : public AGameplayAbilityTargetActor_Trace

```

Comment:
I would inherit from `AGASExtGameplayAbilityTargetActor`

Reply:
It currently inherits from `AGameplayAbilityTargetActor_Trace` but the GASExt class inherits from `AGameplayAbilityTargetActor`. So some functionality is not there in that class which is used here.